### PR TITLE
Make addon work on Firefox 45

### LINKED ISSUE #143: Update Travis-CI to run on FF Beta
From IRC:

> 8:07 AM \<oyiptong\> k88hudson: we should make sure our CI runs beta instead of nightly
> 8:08 AM \<oyiptong\> that's so that we can target release at the next release schedule
> 8:08 AM \<oyiptong\> meaning, we should also fix https://github.com/mozilla/activity-streams/issues/138 in case that only works with nightly/aurora

Not sure if there is an easy way to run a bunch of different browsers in parallel on Travis.
If I understand correctly, we're downloading Nightly on Travis-CI via [**mozdownload**](https://www.npmjs.com/package/mozilla-download) in the [.travis.yml](https://github.com/mozilla/activity-streams/blob/04abb44b36fdb141c8c131d196d73722d4612f15/.travis.yml#L12) file.
Maybe we can define some sort of Travis-CI env matrix where we have can test on beta/aurora/nightly in parallel: https://docs.travis-ci.com/user/speeding-up-the-build/

### PR DESCRIPTION
**Pull request by k88hudson**
Fixes #143

### CODE REVIEW

**Comment by pdehaan** [comment]
I think you also need to change [/karma.conf.js:7](https://github.com/mozilla/activity-streams/blob/gh143/karma.conf.js#L7) (`browsers: ["FirefoxNightly"],`)

And probably modify the [README.md](https://github.com/mozilla/activity-streams/blob/gh143/README.md) which still has a few references to Nightly.

**Comment by pdehaan** [comment]
Currently failing due to the familiar:
```
$ firefox -v
Mozilla Firefox 31.0
```

**Comment by k88hudson** [comment]
:dancer: It's working!

File: .travis.yml [hunk 1]
```diff
@@ -3,15 +3,23 @@ language: node_js
...
   - "export DISPLAY=:99.0"
   - "/sbin/start-stop-daemon --start --quiet --pidfile /tmp/custom_xvfb_99.pid --make-pidfile --background --exec /usr/bin/Xvfb -- :99 -ac -screen 0 1280x1024x16 -extension RANDR"
 
+install:
+  - npm config set spin false
+  - npm install
+
 before_script:
-  - cd $TRAVIS_BUILD_DIR
-  - ./node_modules/.bin/mozilla-download --branch mozilla-central --product firefox firefox
```
Comment (pdehaan) [inline .travis.yml]: Nit: remove unused **mozilla-download** from the [package.json](https://github.com/mozilla/activity-streams/blob/f7eaf012810a5dc7c61bbce10c475aeb29556225/package.json#L71) file.

Comment (k88hudson) [inline .travis.yml]: Ah, good catch

**Comment by k88hudson** [comment]
@pdehaan want to take one more look? I think this should be good to go.

**Comment by pdehaan** [comment]
Works fine for me locally w/ `npm test` and [Firefox 45.0.0 (Mac OS X 10.11.0)] :tada:

File: .travis.yml [hunk 2]
```diff
@@ -3,15 +3,23 @@ language: node_js
...
+
+addons:
+  firefox: latest-beta
+
 before_install:
   - "export DISPLAY=:99.0"
   - "/sbin/start-stop-daemon --start --quiet --pidfile /tmp/custom_xvfb_99.pid --make-pidfile --background --exec /usr/bin/Xvfb -- :99 -ac -screen 0 1280x1024x16 -extension RANDR"
 
+install:
+  - npm config set spin false
```
Comment (pdehaan) [inline .travis.yml]: What does this do here? I thought it was an npm@2 thing only?
I think for npm@3, we disable the progress bar with [https://docs.npmjs.com/misc/config#progress](https://docs.npmjs.com/misc/config#progress) (via the [`--no-progress`](https://github.com/npm/npm/issues/8704#issuecomment-158222250) flag).

**Pull request merged**